### ISSUE TITLE
Why is the input to the model with_missing_mask？

### ISSUE THREAD

**Issue by tangkai-RS** [WenjieDu/SAITS#45]
Hi, great job！ I'm a beginner in the field of time-series imputation. May I ask why the "missing_mask" and "X" concat should be used in the input of SAITS?
``
    def impute(self, inputs):
        X, masks = inputs["X"], inputs["missing_mask"]
        input_X = torch.cat([X, masks], dim=2) if self.input_with_mask else X
        input_X = self.embedding(input_X)
``

**Comment by WenjieDu** [comment]
Hi there,<br><br>Thank you so much for your attention to SAITS! If you find SAITS is helpful to your work, please star⭐️ this repository. Your star is your recognition, which can let others notice SAITS. It matters and is definitely a kind of contribution.<br><br>I have received your message and will respond ASAP. Thank you again for your patience! 😃<br><br>Best,<br>Wenjie

**Comment by WenjieDu** [comment]
NaNs are filled with 0. Concatenating missing_mask can tell the model which values are missing.
